### DISCUSSION TITLE
Question. How do I get the marketplace ID when the authorization happens through SCPN

### DISCUSSION THREAD

**Discussion by DaniZapata** [amzn/selling-partner-api-models#3060] [General]
Hello 

So I got a problem right now making any successful requests to SP-API. Because when I begin the integration by the Partner's third-party network page, we do not get any information about what marketplace the seller belongs to. It is also "impossible" to make requests to SP-API without this information.

I say "impossible"; but there is a solution.
I could make 3 different requests to GET /sellers/v1/marketplaceParticipations changing the marketplaces to see what marketplaces the seller has a presence in and picking the one that returns a 200 response. But What happens when a seller has a presence in North America and Europe?

Has anybody run into this issue?

**Comment by Landsailor** [comment]
There aren't any parameters required for that endpoint, so you should _not_ need to include Marketplace ID. (I can't test it, for some reason, so give it a shot in Postman.) You can filter by using these parameters, though.

Example sandbox response to this call:
`{
						"marketplace" : 
						{
							"countryCode" : "US",
							"defaultCurrencyCode" : "USD",
							"defaultLanguageCode" : "en_US",
							"domainName" : "www.amazon.com",
							"id" : "ATVPDKIKX0DER",
							"name" : "Amazon.com"
						},
						"participation" : 
						{
							"hasSuspendedListings" : false,
							"isParticipating" : true
						}
					}`

**Comment by DaniZapata** [comment]
I am not trying to request GET /sellers/v1/marketplaceParticipations endpoint. I am trying to fetch order information but I can't because for me to get the order information I need the marketplace from where I will be fetching this information, and since I do not know this information natively from Amazon, I'm stuck trying to figure out the marketplace.

**Comment by Landsailor** [comment]
> I am not trying to request [GET /sellers/v1/marketplaceParticipations ](https://developer-docs.amazon.com/sp-api/docs/sellers-api-v1-reference#get-sellersv1marketplaceparticipations) endpoint.

Why not? If your app is approved for Seller Insights and Product Info, you _should_ use that endpoint since it literally [contains the data you're missing](https://developer-docs.amazon.com/sp-api/docs/sellers-api-v1-reference#getmarketplaceparticipationsresponse) to get the orders.

If you're not approved for those roles, then you'll have to hunt and peck or just ask the sellers to provide it.
